source/slang/compiler.cpp

```diff
@@ -976,7 +984,18 @@ String dissassembleDXILUsingDXC(
         TargetRequest*      targetReq,
         UInt                entryPointIndex)
     {
-        auto outputPath = targetReq->entryPointOutputPaths[entryPointIndex];
+        // It is possible that we are dynamically discovering entry
+        // points (using `[shader(...)]` attributes), so that the
+        // number of entry points on the compile request does not
+        // match the number of entries in teh `entryPointOutputPaths`
```

Comment:
the

Reply:
Not an important observation as far as this PR goes, but I notice on looking at this a little more that the 'entryPointIndex' is just the index to the compileRequest entryPoint array. Which makes me wonder why have the index at all - why not place the path in the EntryPointRequest?